README.md

```diff
@@ -105,6 +105,8 @@ Check out the [Quickstart](#quickstart) section for example parameter values.
 
 - **`outdir`**: (Optional) An output location where the Synapse files will be staged. Currently, this location must be an S3 prefix for Nextflow Tower runs. If not provided, this will default to the parent directory of the input file.
 
+- **`simple_stage`**: (Optional) A boolean indicating whether to stage files using a simple folder structure. If `true`, files will be staged in the `outdir`. If `false`, files will be staged in child folders named after the Synapse or Seven Bridges ID of the file. If not provided, this will default to `false`.
```

Comment:
Something to think about instead of a boolean is to use enums. Like a `save_strategy` enum that has:

`DEFAULT`
`SIMPLE`

That way if we add more we don't need a new field.

Reply:
I don't think that Nextflow has a concept like enums. If we wanted multiple different save strategies we could use profiles or more complex parameters. Additionally, the value of `simple_save` or whatever we use has to be a boolean by the time it reaches the process for the conditional `publishDir` setting to work, at least the way I have it implemented now.

Another way would be to set the `publishDir` with a parameter or as part of a profile, but some variables are not accessible within the config file so that could be tricky.

Reply:
I'll leave the final decision up to you how you want to implement this, however, I would consider refactoring it to accept a string and use that for your truthful check later on. This is essentially the same idea as a enum, without calling it an enum. 

The future concern I am thinking about is if we want to add in more ways that we're staging the file we're adding more booleans for those relationships. It makes the interface a bit more complex when there are competing booleans modifying functionality if 2 or more are True.

https://medium.com/geekculture/why-you-shouldnt-use-booleans-in-rest-apis-2747b187876c

Reply:
You make a good point. I will see what I can come up with to allow more options.

Reply:
@BryanFauble See my new changes. I was able to implement the change to allow unlimited `save_strategy`'s and replace `params.simple_stage` with a parameter that accepts a string. I also made the string values more descriptive, with the default value being `id_folders` (files are staged into folders named after the file's ID) and the second supported behavior being called `flat` (files are just dumped into `outdir` with no child folders. As a bonus I got to learn a little bit of Groovy.